dspace-spring-rest/src/main/java/org/dspace/app/rest/IdentifierRestController.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ * <p>
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest;
+
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
+
+import java.net.URI;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.log4j.Logger;
+import org.atteo.evo.inflector.English;
+
+import org.dspace.app.rest.converter.DSpaceObjectConverter;
+import org.dspace.app.rest.link.HalLinkService;
+import org.dspace.app.rest.model.DSpaceObjectRest;
+import org.dspace.app.rest.utils.ContextUtil;
+import org.dspace.content.DSpaceObject;
+import org.dspace.core.Context;
+import org.dspace.identifier.IdentifierNotFoundException;
+import org.dspace.identifier.IdentifierNotResolvableException;
+import org.dspace.identifier.factory.IdentifierServiceFactory;
+import org.dspace.identifier.service.IdentifierService;
+
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.hateoas.Link;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/pid")
+public class IdentifierRestController implements InitializingBean {
+
+    private static final Logger log =
+        Logger.getLogger(IdentifierRestController.class);
+
+    @Autowired
+    DiscoverableEndpointsService discoverableEndpointsService;
+
+    @Autowired
+    private HalLinkService halLinkService;
+
+    @Autowired
+    private List<DSpaceObjectConverter> converters;
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
+        List<Link> links = new ArrayList<Link>();
+
+        Link l = new Link("/api", "pid");
```

Comment:
this should be "/api/pid","pid"
It could be better to define a constant to use in the Controller mapping annotation and here

Reply:
Andrea, I made the change, but I'm not sure what you mean by your comment...